curriculum/challenges/english/16-the-odin-project/top-build-a-recipe-project/top-build-a-recipe-project.md

```diff
@@ -58,13 +58,12 @@ You should have a `h1` element within your `body` element that contains the text
 assert(document.querySelectorAll('BODY > H1')[0].innerText == 'Creamy Chocolate Fudge');
 ```
 
-You should have an image with the url `*placeholder-fcc-cdn*` with an `alt` attribute that has
-a fitting text.
+You should have a fitting image with an `alt` attribute.
```

Comment:
```suggestion
You should have an image with an `alt` attribute.
```

Since the test isn't checking the content of the image, I don't think it's necessary to mention "fitting". If we want something there, I would go with "related" again.